PWGCF/FemtoWorld/Tasks/CMakeLists.txt

```diff
@@ -9,42 +9,20 @@
-          COMPONENT_NAME Analysis)
-
-o2physics_add_dpl_workflow(femto-world-pion-all-pair-track-track
-          SOURCES femtoWorldPionAllPairTask.cxx
-          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
-          COMPONENT_NAME Analysis)
-
-o2physics_add_dpl_workflow(femto-world-pair-track-d0
-          SOURCES femtoWorldPairTaskTrackD0.cxx
-          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
-          COMPONENT_NAME Analysis)
-
-o2physics_add_dpl_workflow(femto-world-pair-pion-pion
-          SOURCES femtoWorldPairTaskPionPion.cxx
-          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
-          COMPONENT_NAME Analysis)
-
-o2physics_add_dpl_workflow(femto-world-pair-proton-antiproton
-          SOURCES femtoWorldPairTaskProtonAntiproton.cxx
-          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
-          COMPONENT_NAME Analysis)
-
-o2physics_add_dpl_workflow(femto-world-efficiency-task
-          SOURCES femtoWorldEfficiencyTask.cxx
-          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
-          COMPONENT_NAME Analysis)
\ No newline at end of file
+o2physics_add_dpl_workflow(femto - world - pair - track - track SOURCES femtoWorldPairTaskTrackTrack.cxx PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore COMPONENT_NAME Analysis)
+
+  o2physics_add_dpl_workflow(femto - world - pair - track - phi SOURCES femtoWorldPairTaskTrackPhi.cxx PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore COMPONENT_NAME Analysis)
+
+    o2physics_add_dpl_workflow(femto - world - pion - pair - track - track SOURCES femtoWorldPionPairTaskTrackTrack.cxx PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore COMPONENT_NAME Analysis)
+
+      o2physics_add_dpl_workflow(femto - world - pion - all - pair - track - track SOURCES femtoWorldPionAllPairTask.cxx PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore COMPONENT_NAME Analysis)
+
+        o2physics_add_dpl_workflow(femto - world - pair - track - d0 SOURCES femtoWorldPairTaskTrackD0.cxx PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore COMPONENT_NAME Analysis)
+
+          o2physics_add_dpl_workflow(femto - world - pair - pion - pion SOURCES femtoWorldPairTaskPionPion.cxx PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore COMPONENT_NAME Analysis)
+
+            o2physics_add_dpl_workflow(femto - world - pair - proton - antiproton SOURCES femtoWorldPairTaskProtonAntiproton.cxx PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore COMPONENT_NAME Analysis)
+
+              o2physics_add_dpl_workflow(femto - world - efficiency - task SOURCES femtoWorldEfficiencyTask.cxx PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore COMPONENT_NAME Analysis)
+
+                o2physics_add_dpl_workflow(femto - world - efficiency - task - de SOURCES femtoWorldEfficiencyTaskDe.cxx PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore COMPONENT_NAME Analysis)
```

Comment:
Why are you getting this formatting?
This will not compile I guess

Reply:
There was a formatting mistake during GitHub check procedure, I used clang utility to solve it automatically; however, since you suppose that it should not compile, I shall bring it back to the way it was originally.